### PR TITLE
Use 10.9 SDK with new libc++

### PR DESCRIPTION
**Pull request by isuruf**
Fixes https://github.com/conda-forge/cmake-feedstock/issues/99

See the comment in the code.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: .ci_support/osx_.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.14'
+- '10.9'
```
Comment (jakirkham) [inline .ci_support/osx_.yaml]: Interesting. Why was 10.14 used here?

Comment (isuruf) [reply .ci_support/osx_.yaml]: See the discussion in https://github.com/conda-forge/cmake-feedstock/pull/98

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
This is ready for review

File: .ci_support/osx_.yaml (hunk 1 above)
Comment (jakirkham) [reply .ci_support/osx_.yaml]: Thanks for the context. Also nice work on finding this solution! 😄

**Pull request merged**